.github/workflows/beta-release.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish beta release when pushing to the main branch
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'npm'
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - run: npm ci
+      - run: npm run build
+
+      # Note, that the angular library needs to be built again to update the package.json in
+      # 'libraries/ui-library-angular/dist/ui-library-angular/package.json'.
+      - name: bump version to next beta release
+        run: |
+          npm version prerelease -w libraries --no-git-tag-version
```

Comment:
If we create no tag we would have to define another way to show what has changed since the last beta release in the changelog, if we decide to show one between beta releases.

Reply:
as discussed, lets merge this now and find a solution for the tagging and changelog-generation leater